assets/js/admin.js

```diff
@@ -166,6 +166,155 @@
 		// Social
 		$( document ).on( 'click', '.wpcw-widget-social .icons a', socialField.init );
 
+		$( 'body' ).on( 'click', '.day-container', function() {
+
+			var container = $( this );
+
+			container.find( 'div.hidden-container' ).slideToggle( 'fast', function() {
+
+				if ( container.hasClass( 'closed' ) ) {
+
+					container.removeClass( 'closed' ).addClass( 'open' );
+
+					return;
+
+				}
+
+				container.removeClass( 'open' ).addClass( 'closed' );
+
+			} );
+
+		} );
+
+		$( 'body' ).on( 'click', '.add-time', function( e ) {
+
+			var parent_container = $( this ).parents( '.hidden-container' ).find( '.hours-selection' ).last(),
+          clone            = parent_container.clone();
+
+			clone.find( 'select[name*="[open]"], select[name*="[closed]"]' ).attr( 'name', function( i, name ) {
+
+				return name.replace( /\[(\d+)\]$/, function( match, number ) {
+
+					return '[' + ( + number + 1 ) + ']';
+
+				} );
+
+			});
+
+			clone.find( '.add-time' ).replaceWith( '<a href="#" class="remove-time button-secondary"><span class="dashicons dashicons-no-alt"></span></a>' );
+
+			clone.insertAfter( parent_container );
+
+			e.preventDefault();
+
+		} );
+
+		$( 'body' ).on( 'click', '.remove-time', function( e ) {
+
+			var button = $( this ),
+			    parent = button.parent( '.hours-selection' );
+
+			parent.fadeOut( 'fast', function() {
+
+				parent.remove();
+
+			} );
+
+			e.preventDefault();
+
+		} );
+
+		$( 'body' ).on( 'click', '.js_wpcw_closed_checkbox, .wpcw-widget-hours select, .add-time, .remove-time, .js_wpcw_apply_hours_to_all', function( e ) {
+
+			e.stopPropagation();
+
+		} );
+
+		// Hours of Operation select field toggle
+		$( 'body' ).on( 'change', '.js_wpcw_closed_checkbox', function( e ) {
+
+			var select_fields = $( e.currentTarget ).parents( '.day-container' ).find( 'select' );
```

Comment:
@EvanHerman As a good rule of thumb, vars that represent element objects should use a `$` prefix, while vars that simple contain data should not. This can be helpful when examining the code later.